packages/cairomm_1_0.rb

```diff
@@ -0,0 +1,45 @@
+require 'package'
+
+class Cairomm_1_0 < Package
```

Comment:
Wouldn't a better name be `Cairomm_1_14`?

Reply:
no. The library name is cairomm-1.0. This is important because packages demand `cairomm-1.0`.

Debian has it broken out the same way for I assume the same reason: https://packages.debian.org/sid/libcairomm-1.0-1v5